packages/common/src/ui/components/navigation/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,24 @@
+import { Box, BoxProps } from '@material-ui/system';
+import React, { FC } from 'react';
+
+type TabPanelProps = {
+  currentTab: number;
+  tab: number;
+} & BoxProps;
+
+export const TabPanel: FC<TabPanelProps> = ({
+  children,
+  tab,
+  currentTab,
+  ...boxProps
+}) => {
+  return (
+    <>
+      {currentTab === tab && (
+        <Box role="tabpanel" {...boxProps}>
+          {children}
+        </Box>
+      )}
+    </>
+  );
```

Comment:
spreading the box props in this one and continuing to be inconsistent. Thought it would be annoying to not be able to add margins if you needed to add margins to your page etc..